plugins/source/aws/resources/services/cloudformation/stacks_mock_test.go

```diff
@@ -40,6 +41,21 @@ func buildStacks(t *testing.T, ctrl *gomock.Controller) client.Services {
 		nil,
 	)
 
+	var summary cloudformation.GetTemplateSummaryOutput
+	if err := faker.FakeObject(&summary); err != nil {
+		t.Fatal(err)
+	}
+	summary.Metadata = aws.String("{}") // TODO this feels hacky - what is a good solution here?
```

Comment:
Note, this 'fixes' the failing test - an issue with the JSONinfication of the field when it is faked during this test -but I'm not sure I understand things well enough here. There must be a better solution?! Input welcome.